components/organisms/TopNav/top-nav.tsx

```diff
@@ -85,19 +79,17 @@ const Nav = ({ className }: { className?: string }) => {
             Highlights
           </Link>
         </li>
-        {state.value && (
-          <li className="hidden lg:inline">
-            <Link
-              className={`tracking-tight font-medium text-sm text-slate-700 hover:text-orange-500 flex gap-1 items-center transition-all ${getActiveStyle(
-                router.pathname === "/star-search"
-              )}`}
-              href={"/star-search"}
-            >
-              <img src="/assets/star-search-logo.svg" alt="Star Search logo" className="w-5 h-5" />
-              <p className="mt-0.5">StarSearch</p>
-            </Link>
-          </li>
-        )}
+        <li className="hidden lg:inline">
```

Comment:
The feature flag check for the StarSearch link is no longer required as we have a waitlist page.